### PR TITLE
Adds more tests to jclouds module.

### PR DESCRIPTION
**Pull request by kobalski**

### CODE REVIEW

File: hazelcast-jclouds/src/test/java/com/hazelcast/jclouds/ComputeServiceBuilderTest.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            nodes.add(new NodeMetadataBuilder().id(UuidUtil.newSecureUuidString()).group("group" + i)
+                    .status(NodeMetadata.Status.RUNNING).build());
+        }
+        nodes.add(new NodeMetadataBuilder().id(UuidUtil.newSecureUuidString()).group(groupUnderTest)
+                .status(NodeMetadata.Status.RUNNING).build());
+        doReturn(nodes).when(mockComputeService).listNodesDetailsMatching(null);
+
+        Set<NodeMetadata> result = (Set) builder.getFilteredNodes();
+
+        assertEquals(2, result.size());
```
Comment (mesutcelik) [inline hazelcast-jclouds/src/test/java/com/hazelcast/jclouds/ComputeServiceBuilderTest.java]: you add 5 but you assert 2... I couldn't get the logic?

**Comment by mesutcelik** [comment]
:+1:

File: hazelcast-jclouds/src/test/java/com/hazelcast/jclouds/ComputeServiceBuilderTest.java (hunk 1 above)
Comment (kobalski) [inline hazelcast-jclouds/src/test/java/com/hazelcast/jclouds/ComputeServiceBuilderTest.java]: I add 5 with increasing group number, `group1`,`group2` etc and add one extra `group3` outside of the for loop and assert `builder.getFilteredNodes ` filters nodes correctly for `group3` which should be 2 nodes.

**Comment by serkan-ozal** [comment]
:+1:

**Pull request merged**